### ISSUE TITLE
Increase contrast ratio for markers via default styling

### ISSUE THREAD

**Issue by danny-andrews** [reach/reach-ui#670]
### Current Behavior

The background color of the track and markers is [`#f2f2f2`](https://github.com/reach/reach-ui/blob/develop/packages/slider/styles.css#L28). This makes these elements difficult to distinguish on a white background. The contrast ratio is a measly 1.11:1! 

![image](https://user-images.githubusercontent.com/14965732/93526248-cc39ab80-f8fc-11ea-93fd-a6e29a73bf80.png)

### Expected behavior

The track and markers should be styled to be more easily distinguishable on a white background. Playing around on https://webaim.org/resources/contrastchecker/, I came up with `#D4D4D4`, which gives a contrast ratio of ~1.5:1.

![image](https://user-images.githubusercontent.com/14965732/93526774-8af5cb80-f8fd-11ea-93e7-c1068d8495c7.png)

I realize the default styles are merely there to get a functional component, and are meant to be customized, but the default should at least be accessible.

**Comment by chaance** [comment]
I've since overhauled the slider's default styles quite a bit, but I'll double check to make sure this passes contrast checks. Thank you!
